otter/auth.py

```diff
@@ -235,6 +235,13 @@ def __init__(self, identity_admin_user, identity_admin_password, url, admin_url)
 
     @wait(ignore_kwargs=['log'])
     def _auth_me(self, log=None):
+        def _log_failed_auth(err):
+            """
+            Log this as a string we know we can find in the logging feed
+            """
+            log.err(err, 'Failed to get a new identity admin token.', otter_msg_type='impersonation-failed')
```

Comment:
Line too long, that's why the build's failing. sorry :P lint now expects <80 column lines.